src/main/kotlin/de/tfr/impf/selenium/DriverFactory.kt

```diff
@@ -5,18 +5,67 @@ import org.openqa.selenium.WebDriver
 import org.openqa.selenium.chrome.ChromeDriver
 import org.openqa.selenium.chrome.ChromeOptions
 import java.util.concurrent.TimeUnit
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement
 
 fun createDriver(): ChromeDriver {
     System.setProperty(Config.nameDriver, Config.pathDriver + Config.exeDriver)
     val chromeOptions = ChromeOptions()
     if (Config.hasUserAgent) {
         chromeOptions.addArguments("general.useragent.override", Config.userAgent)
     }
+
     val chromeDriver = ChromeDriver(chromeOptions)
     chromeDriver.setTimeOut(Config.searchElementTimeout())
+
+    chromeDriver.get("chrome://settings/clearBrowserData")
```

Comment:
A little bit hacky for just restarting a new browser session.
Have you also tried it with something like this?
```java
public static void restartDriver() {
    driver.manage().deleteAllCookies();         // Clear Cookies on the browser
    driver.quit();
    driver = new ChromeDriver();
}
```
👉 https://stackoverflow.com/a/51521182/4198170

Or this one, also seems to do the right thing:
```java
   private void clearDriverCache(ChromeDriver driver) {
    driver.getDevTools().createSessionIfThereIsNotOne();
    driver.getDevTools().send(Network.clearBrowserCookies());
}
```
👉 https://stackoverflow.com/a/60592482/4198170

Reply:
@TobseF there is a difference between cookie clear and cookie&cache. some browser finterprinter will check for already cached files. Unfortunately there is no easy solution: https://www.reddit.com/r/selenium/comments/aok960/can_no_longer_clear_chrome_cache_via_css_selector/

Reply:
OK.. but also clearing the cache seems to be possible with the DevTools:

https://stackoverflow.com/a/55636217/4198170
https://chromedevtools.github.io/devtools-protocol/tot/Network/#method-clearBrowserCache

But anyway... if your setup already helps... I will merge it.

Reply:
Actually the merge of this pull request didn't help at all, quite the opposite - it broke my setup. Selenium couldn't find root5 ("settings-clear-browsing-data-dialog"). That's why I switched back to e34d36e.

I could reproduce the issue on two Windows 10 PCs. Anyway... maybe I'm the only one. @TobseF , thank you very much for your effort!

Reply:
@fkab Thanks for your hint. I separated clear browser data and clear cache. Both can be enabled by these settings:
```properties
# Clears browser cookies to avoid bot detection. Disabled by default.
clearCookies = false
# Experimental feature, which tries to clear the browser cache to avoid bot detection. Disabled by default.
experimentalClearBrowser = false
```
So it shouldn't crash your setup anymore.